doc/sphinx/user-guide.rst

```diff
@@ -116,7 +116,7 @@ is performed using the `List Blobs API <List Blobs API_>`_. For example
         
         .. literalinclude:: ../../samples/map_dataset/dataset_from_container_url.py
             :lines: 9-
-
+    
```

Comment:
It looks like there is still some whitespace being added? To tell it is actually removed, I'd recommend running `git diff` against main to make sure this file does not change. In general, it would not hurt to run `git diff` prior to pushing changes to make sure all of the changes are accounted for or there is no unwanted changes.